recipe/meta.yaml

```diff
@@ -24,8 +18,6 @@ requirements:
     - cmake
     - git
```

Comment:
I'm not sure if this is still a thing, but we may need: 
```suggestion
    - git   # [not win]
```

Reply:
git can probably be removed entirely, as we are not using git_url in the source anymore.